examples/rollbar-spring-webmvc/build.gradle

```diff
@@ -0,0 +1,45 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'com.bmuschko:gradle-tomcat-plugin:2.3'
+    }
+}
+
+ext {
+    springWebmvcVersion = '5.2.6.RELEASE'
+    javaxServletVersion = '3.0.1'
+    junitVersion = '4.12'
+    mockitoVersion = '2.10.0'
+    springFrameworkBootTestVersion = '2.2.6.RELEASE'
+}
+
+apply plugin: 'war'
+apply plugin: 'com.bmuschko.tomcat'
+
+dependencies {
+    compile project(":rollbar-web")
+    compile project(":rollbar-spring-webmvc")
+
+    providedCompile 'javax.servlet:javax.servlet-api:3.1.0'
```

Comment:
This dependency and https://github.com/rollbar/rollbar-java/pull/215/files#diff-e3ac346b0a481e8d360ca335ee6b8604R29 I think are not needed.
Also, neither the ones for `testCompile` as we are not implementing tests of the example application.